features/old/authorization/provider_finance.feature

```diff
@@ -31,7 +31,7 @@ Feature: Provider finance authorization
       And I go to the provider dashboard
     Then I should not see "Billing" in the audience dashboard widget
 
-    When I request the url of the '<page>' page then I should see an exception
+    When I request the url of the <page> page then I should see an exception
```

Comment:
It's fine to keep as is for the purposes of this pull request. Just a thought for the future that such step might be renamed to something like `opening <page> page results in an exception` or something along these lines.